src/Hastlayer/Hast.Communication/Services/FpgaIpEndpointFinder.cs

```diff
@@ -28,10 +29,13 @@ public FpgaIpEndpointFinder(IClock clock, ILogger<FpgaIpEndpointFinder> logger)
         _logger = logger;
     }
 
+    [SuppressMessage(
+        "Style",
+        "IDE0230:Use UTF-8 string literal",
+        Justification = "False positive, shouldn't replace the char constant with a literal.")]
```

Comment:
This is flagged as unnecessary for me.

Reply:
VS being garbage again.

Reply:
![image](https://github.com/Lombiq/Hastlayer-SDK/assets/1976647/a81552f3-4723-4ea7-93b4-90bfd59e8496)